src/pages/clients/edit/Edit.tsx

```diff
@@ -120,10 +120,6 @@ export default function Edit() {
           toast.dismiss();
           setErrors(error.response.data);
         }
-
-        if (error.response?.status === 412) {
-          toast.error('password_error_incorrect');
```

Comment:
@beganovich This if statement for the "/api/v1/clients/:id" endpoint is unnecessary since it doesn't require the "X-Api-Password" header.